pkg/config/config.go

```diff
@@ -110,6 +117,17 @@ func (rule *Rule) UnmarshalJSON(data []byte) error {
 		return errLevelMustBeString
 	}
 
+	if ignore, ok := result["ignore"]; ok {
+		var dst Ignore
+
+		err := rio.JSONRoundTrip(ignore, &dst)
+		if err != nil {
+			return fmt.Errorf("unmarshalling rule ignore failed: %w", err)
+		}
+
+		rule.Ignore = dst
+	}
+
 	delete(result, "enabled")
```

Comment:
Wonder if we should delete "level" and "ignore" as well, or is there a particular reason this isn't already done in the case of "level"?

Reply:
"enabled" is what we used to call it (and it was a boolean value) before we changed to "level" to allow more granual settings, so this should be dead code. I'll have it removed in a followup PR.